docs/card.md

```diff
@@ -216,6 +214,27 @@ specify styling for view surrounding image
 
 ---
 
+### `imageOnPress`
+
+trigger Header Image press (optional)
+
+|   Type   | Default |
+| :------: | :-----: |
+| function |  none   |
+
+---
+
+### `imageOnPressColor`
```

Comment:
Instead of passing a color here, it might be best to have a property called `imageOnPressProps` (or a better name) so that the user can pass any props from `TouchableHighlight`

Reply:
Great idea! it's done